pkg/virt-launcher/virtwrap/network/podnic_test.go

```diff
@@ -80,7 +80,7 @@ var _ = Describe("podNIC", func() {
 			Expect(ok).To(BeTrue(), "SetupPhase1 should return an error of type CriticalNetworkError")
 		})
 	})
-	Context("when DHCP startup fails", func() {
+	PContext("when DHCP startup fails", func() {
```

Comment:
I'm not sure "pending" is the right thing, but yes... Both [P](https://github.com/onsi/ginkgo/blob/master/ginkgo_dsl.go#L409) and [X](https://github.com/onsi/ginkgo/blob/master/ginkgo_dsl.go#L415) prefixes mean "pending" to gingkgo, so I guess they're both equally right / wrong.

Reply:
I believe that mocking the `dhcpConfigurator` should solve the issue.